src/pages/Game/hooks/usePlayerKeys.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect } from 'react';
+
+import { MIN_SPEED } from '../constants';
+
+import { Layer, PlayerAction, Shortcut } from '../types';
+
+type KeysMap = Record<PlayerAction, Shortcut>;
+
+export const usePlayerKeys = (player: Layer, keysMap: KeysMap) => {
+  useEffect(() => {
+    const onKeyDown = (event: KeyboardEvent) => {
+      const { code } = event;
+
+      if (code === keysMap.moveLeft.code) {
+        player.setVx(-MIN_SPEED);
```

Comment:
🧐
Так более наглядно, унарный минус можно проглядеть
```suggestion
        player.setVx(MIN_SPEED * -1);
```

Reply:
Поправил.